presto-manager-common/src/main/java/com/teradata/prestomanager/common/ApiRequester.java

```diff
@@ -102,6 +102,28 @@ private Invocation createInvocation(URI uri)
         return invocation;
     }
 
+    private static class UriCopyBuilder
+    {
+        private final UriBuilder template;
+
+        private UriCopyBuilder(UriBuilder template)
+        {
+            this.template = requireNonNull(template);
```

Comment:
The message for `requireNonNull` would help debug the problem in case null was passed here.